src/test/resources/org/prebid/server/ApplicationTest/test-application.properties

```diff
@@ -135,3 +135,4 @@ analytics.log.enabled=true
 gdpr.host-vendor-id=1
 gdpr.vendorlist.filesystem-cache-dir=src/test/resources/org/prebid/server/ApplicationTest/gdpr-vendorlist
 gdpr.geolocation.maxmind.db-archive-path=/org/prebid/server/geolocation/GeoLite2-test.tar.gz
+events.accounts-enabled=1001
```

Comment:
Let's improve one of the /openrtb2/auction and /openrtb2/amp test in AppliationTest to verify functionality implemented